apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -273,7 +273,8 @@ export const UserForm = ({
         </FormGroup>
       )}
       {isFeatureEnabled(Feature.DeclarativeUserProfile) &&
-      realm?.attributes?.userProfileEnabled === "true" ? (
+      realm?.attributes?.userProfileEnabled !== undefined &&
+      realm.attributes.userProfileEnabled === "true" ? (
```

Comment:
The first statement here is redundant. `realm?.attributes?.userProfileEnabled === "true"` will already implicitly assert that `realm.attributes.userProfileEnabled` is not `undefined`.

So unless I am missing something the following code should yield the same result.
```suggestion
      realm?.attributes?.userProfileEnabled === "true" ? (
```